### PR TITLE
Cudamapper check input files fix309

### PR DESCRIPTION
**Pull request by rached-ab**

### CODE REVIEW

File: common/io/src/kseqpp_fasta_parser.cpp [hunk 1]
```diff
@@ -28,16 +30,34 @@ FastaParserKseqpp::FastaParserKseqpp(const std::string& fasta_file, int min_sequ
 {
     klibpp::KSeq record;
     klibpp::SeqStreamIn iss(fasta_file.data());
+
+    /*    std::ifstream query_file(fasta_file);
```
Comment (tijyojwad) [inline common/io/src/kseqpp_fasta_parser.cpp]: @rached-ab let's remove the commented out code

Comment (rached-ab) [reply common/io/src/kseqpp_fasta_parser.cpp]: @tijyojwad do you think it is needed to add try/catch in non regression tests when parser in created ?

File: cudamapper/src/main.cu [hunk 2]
```diff
@@ -289,17 +289,33 @@ int main(int argc, char* argv[])
     claragenomics::logging::Init();
 
     const ApplicationParameteres parameters = read_input(argc, argv);
+    std::shared_ptr<claragenomics::io::FastaParser> query_parser;
 
-    std::shared_ptr<claragenomics::io::FastaParser> query_parser = claragenomics::io::create_kseq_fasta_parser(parameters.query_filepath, parameters.k + parameters.w - 1);
-
+    try
```
Comment (ahehn-nv) [inline cudamapper/src/main.cu]: While we should probably catch all exceptions in main, I would not do this for individual function calls.
For now I would rely on the C++ implementation to have a good handler for all uncaught exceptions which prints the error for us and ends the program.

Comment (rached-ab) [reply cudamapper/src/main.cu]: done.

**Comment by tijyojwad** [comment]
Closes #309

**Pull request merged**